plugins/unocss.ts

```diff
@@ -0,0 +1,29 @@
+import type { Preset, UserConfig } from 'https://esm.sh/@unocss/core@0.53.1'
+import { UnoGenerator } from 'https://esm.sh/@unocss/core@0.53.1'
+import presetUno from 'https://esm.sh/@unocss/preset-uno@0.46.3?bundle&no-check'
+import { Plugin } from '$fresh/server.ts'
+
+const unoResetCSS = `/* reset */
```

Comment:
Could you add a comment where you're getting this from?

Reply:
It's originally from [here](https://github.com/Hladikes/fresh/blob/main/plugins/unocss.ts) but not sure which exact reset is being applied from the [official unocss resets](https://unocss.dev/guide/style-reset). Happy to this...

Reply:
Not gonna lie, it feels great to have my nickname in comments since this is my first sort of a contribution. However I'm not the original author. Here is the [UnoCSS plugin](https://github.com/ije/html/blob/main/plugins/unocss.ts) by [ije](https://github.com/ije), which I took and slightly modified, and in which he commented the name of the used preset:) Take a look at [here](https://github.com/ije/html/blob/6c5612fb81e5159c1a598eb6b37a3f4c72a25fe5/plugins/unocss.ts#L6C52-L6C52)

Reply:
Ha! Glad to have you around @Hladikes ! Thanks for the pointer, I will update the comment to point to `https://esm.sh/@unocss/reset@0.51.2/tailwind.css` instead 👍🏼